Add jdeb support where missing

nareshgbhat opened the pull request:
This patch add debian package creation support for utility package.

Signed-off-by: Naresh Bhat <naresh.bhat@linaro.org>

## What changes were proposed in this pull request?

Add jdeb support to utility component.

## How was this patch tested?

Test on Debian-9 installed on AArch64 (branch - release-2.5.2).

Please review [Ambari Contributing Guide](https://cwiki.apache.org/confluence/display/AMBARI/How+to+Contribute) before opening a pull request.

asfgit commented:
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/Ambari-Github-PullRequest-Builder/2369/
Test FAILed.
Test FAILured.

adoroszlai commented on utility/pom.xml:
```diff
@@ -58,6 +58,26 @@
 
   <build>
     <plugins>
+        <groupId>org.vafer</groupId>
```
`<plugin>` is missing here, causing:

```
[FATAL] Non-parseable POM utility/pom.xml: Unrecognised tag: 'groupId' (position: START_TAG seen ...<plugins>\n        <groupId>... @61:18)  @ line 61, column 18
```

nareshgbhat replied:
My apologies for late reply.  Sure, I will make the necessary changes and give a single pull request.  Thank you very much for suggestions.

adoroszlai reviewed (changes requested):
Hi @nareshgbhat, thanks for the contribution.

Can you please incorporate all jdeb-related changes (#1319, #1320, #1322) in this single pull request?  The rationale is that they can only be tested together.  Running `jdeb:jdeb` on individual modules results in:

```
[INFO] --- jdeb:1.0.1:jdeb (default-cli) @ utility ---
[INFO] skipping sub module: jdeb executing at top-level only
```

nareshgbhat committed:
ambari: Add jdeb support

This patch add jdeb (debian) package creation support for the following
packages.
-ambari-funtest
-ambari-logsearch
-ambari-metrics-grafana
-utility

Signed-off-by: Naresh Bhat <naresh.bhat@linaro.org>

asfgit commented:
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/Ambari-Github-PullRequest-Builder/2478/
Test FAILed.
Test FAILured.

nareshgbhat commented:
@adoroszlai  The build failure does not looks like related to changes in my patch.  If so please let me know your suggestions. Thanks.

asfgit commented:
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/Ambari-Github-PullRequest-Builder/2479/
Test FAILed.
Test FAILured.

nareshgbhat commented:
I also want to apply the same patch for the branch release-2.6.1 and release-2.6.2 should I create a separate jira issue and give the pull request or can I use the same #1323  thread ?

adoroszlai commented:
@nareshgbhat You don't need a seperate Jira issue, but a new pull request is required, since pull requests are specific to a target branch.  Also, `release-2.6.1` and `release-2.6.2` are tags -- you can apply similar change to `branch-2.6` instead.

nareshgbhat commented:
@adoroszlai Thanks I have raised one more pull request for the same patch on v2.6 branch - https://github.com/apache/ambari/pull/1417

asfgit commented:
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/Ambari-Github-PullRequest-Builder/2521/
Test FAILed.
Test FAILured.

asfgit commented:
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/Ambari-Github-PullRequest-Builder/2520/
Test FAILed.
Test FAILured.

asfgit commented:
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/Ambari-Github-PullRequest-Builder/2522/
Test FAILed.
Test FAILured.

asfgit commented:
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/Ambari-Github-PullRequest-Builder/2523/
Test FAILed.
Test FAILured.

adoroszlai commented:
Hi @nareshgbhat, I just noticed that #1417 was opened from the same source branch in your repository, and that branch `AMBARI-23903` is now based on `branch-2.6`.  This causes that this pull request cannot be merged (since we don't want to merge hundreds of commits into `branch-2.5`.

Can you please create a new branch based on `branch-2.5`, apply your changes and update this pull request?

In general, if you are planning to apply a change to multiple branches (eg. `trunk` and `branch-2.6`), then a separate source branch and a separate pull request is needed for each target.  They can all reference the same Jira issue, though.  It is common to name branches using both Jira number and target branch name, eg. `AMBARI-23903_branch-2.5`, etc.

nareshgbhat commented:
Done. My apologies for inconvenience.

asfgit commented:
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/Ambari-Github-PullRequest-Builder/2599/
Test FAILed.
Test FAILured.

nareshgbhat commented:
@adoroszlai Thanks for merging the patch.  I have noticed that the Author is different in the merged patch

Author: Naresh Bhat <nareshgbhat@users.noreply.github.com>

it should be

Author: Naresh Bhat <naresh.bhat@linaro.org>

Is it possible to update the patch with git author command ?  I am not sure how it got modified.
and also I have added description and signoff which is missing in the patch.  The patch must have looked as below,

Author: Naresh Bhat <naresh.bhat@linaro.org>
Date:   Tue May 29 14:31:39 2018 +0530

    ambari: Add jdeb support
    
    This patch add jdeb (debian) package creation support for the following
    packages.
    -ambari-funtest
    -ambari-logsearch
    -ambari-metrics-grafana
    -utility
    
    Signed-off-by: Naresh Bhat <naresh.bhat@linaro.org>

...
..........

After adding description, signoff.  Is it possible to cherry-pick the same patch on v2.6 branch ?

adoroszlai commented:
To add it to `branch-2.6`, please cherry-pick locally, and open a new PR.

The noreply address is used if "Keep my email address private" is enabled in your settings.  I can try choosing "Rebase and merge" next time and see if it keeps the original commit's email address.

nareshgbhat commented:
@adoroszlai Thank you very much.  This is my first patch using github. I will try to avoid mistakes while doing further contribution.  I have unchecked the "Keep my email address private" box.  I will cherry-pick locally and open a new PR.
